tests/run-make/rust-lld/rmake.rs

```diff
@@ -60,7 +60,8 @@ fn main() {
 }
 
 fn find_lld_version_in_logs(stderr: String) -> bool {
-    let lld_version_re =
-        Regex::new(r"^warning: linker std(out|err): LLD [0-9]+\.[0-9]+\.[0-9]+").unwrap();
+    // Strip the `-Wlinker-messages` wrappers prefixing the linker output.
+    let stderr = Regex::new(r"warning: linker std(out|err):").unwrap().replace_all(&stderr, "");
+    let lld_version_re = Regex::new(r"^LLD [0-9]+\.[0-9]+\.[0-9]+").unwrap();
```

Comment:
Maybe it would be enough to use a single regex `Regex::new(r"^.*LLD [0-9]+\.[0-9]+\.[0-9]+").unwrap()`

Reply:
Hmm. It's specifically looking for a line version in the output, not a prefix, otherwise it could be searching the entire stderr. I did it this way because I found it likely the message wrapper stripping will be moved in `run_make_support` once it's solidified, esp if we encounter more searching in linker output in the future. So while I'm not particularly convinced, I'm also not too invested in any approach either.